src/pint/models/jump.py

```diff
@@ -136,3 +136,94 @@ def get_jump_param_objects(self):
         """
         jump_obs = [getattr(self, jump) for jump in self.jumps]
         return jump_obs
+
+    def jump_params_to_flags(self, toas):
+        """Take jumps created from .par file and add appropriate flags to toa table.
+
+        This function was made specifically with pintk in mind for a way to properly
+        load jump flags at the same time a .par file with jumps is loaded (like how
+        jump_flags_to_params loads jumps from .tim files).
+
+        Parameters
+        ----------
+        toas: TOAs object
+            The TOAs which contain the TOA table to be modified
+        """
+        # for every jump, set appropriate flag for TOAs it jumps
+        for jump_par in self.get_jump_param_objects():
+            # find TOAs jump applies to
+            mask = jump_par.select_toa_mask(toas)
+            # apply to dictionaries
+            for dict in toas.table["flags"][mask]:
+                if "jump" in dict.keys():
+                    # check if jump flag already added - don't add flag twice
+                    if jump_par.index in dict["jump"]:
+                        continue
+                    dict["jump"].append(jump_par.index)  # otherwise, add jump flag
+                else:
+                    dict["jump"] = [jump_par.index]
+
+    def add_jump_and_flags(self, toa_table):
+        """Add jump object to PhaseJump and appropriate flags to TOA tables (helper
```

Comment:
This should be just one line. Maybe move the "helper function" to a separate line, below? This will be shown in brief method listings.